usr/sbin/registercloudguest

```diff
@@ -540,6 +556,15 @@ if registration_returncode != 0:
 else:
     print('Registration succeeded')
 
+if failed_extensions:
+    print(
+        'There are products that were not installed because they need '
+        'an additional registration code, to install them please call'
```

Comment:
Not sure if "call" is a universal term here. Suggest something like "run the following command:".

Reply:
Done